.github/CODEOWNERS

```diff
@@ -1,15 +1,25 @@
-# default code-owner
+# Default code-owner
 * @ethereum-optimism/security-reviewers
 
+# Code and script ownership
+
+## Go
 *.go @ethereum-optimism/go-reviewers
 /go.work @ethereum-optimism/go-reviewers
 /.golangci.yml @ethereum-optimism/go-reviewers
 /validation @ethereum-optimism/go-reviewers
 /superchain @ethereum-optimism/go-reviewers
-/superchain/extra/addresses @ethereum-optimism/security-reviewers
 
+## JS&TS
 *.js @roninjin10
 package.json @roninjin10
```

Comment:
We should probably put someone else here since @roninjin10 will be leaving soon 😢

We have a `package.json`, 1 JS file, no TS files, and no deps. The main thing in the `package.json` is an alias for running the codgen script. Potential solution:
- Run the codegen script as part of `add-chain.sh` (cc @geoknee)
- Then we can remove the `package.json` (or do we currently use this for npm releases @roninjin10?)
- Make security the codeowner for TS/JS for now

Reply:
Sorry, I don't think I see the security JS/TS codeowner line?

Reply:
Ah I was looking for JS/TS explicitly. This works, resolving!

Reply:
We need package.json so you can install superchain repo like `npm install https://github.com/ethereum-optimism/superchain-registry/tree/main/superchain/configs/goerli`. It also is used for npm releases.

I think security team can get by owning it just fine as I wouldn't expect the package.json to change much if ever.

If we wanted a js person to still own it @nitaliano is best bet

Reply:
correction the only change to it is bumping the version number if releasing a new version to npm.

It's perfectly reasonable IMO to expect people to install from github though like the command I put above

Reply:
Got it! Do you know if any projects that install this repo in that way? And presumably the idea is they can then just import this repo's JSON files into JS/TS projects? (The reason for asking is tangential to this PR—we were looking for that type of usage tohelp us with our effort to find all current consumers https://www.notion.so/oplabs/Superchain-Registry-Consumers-19c4e715c75341db813f3fed37e03e5c?pvs=4)

Re publishing to npm, I see we have a `publish-bot` job in CI but it seems unused

Reply:
Yea it's actually pretty common on blockchain industry specifically. Before we switched to using foundry submodules optimism used github installations to install solidity contract packages that were otherwise not.

Realistically most JavaScript consumers will be getting constants like contract addresses from a higher level library like `viem` or `tevm/opstack` as well.

Reply:
Ok got it, thank you! I've noted this in the above notion doc and will resolve this comment now